### ISSUE TITLE
Dynamics CRM can't connect

### ISSUE THREAD

**Issue by samuelbirch** [verbb/formie#682]
Hi,

We're having trouble connecting to Dynamics CRM, once all the details have been entered and saved clicking on "connect" gives us back this error:

“Dynamics CRM” response: “Required parameter not passed: "code"”

**Comment by engram-design** [comment]
This is usually an issue with the OAuth handshake. Just wanted to triple-check you've followed https://verbb.io/craft-plugins/formie/docs/integrations/crm-integrations#microsoft-dynamics-365 (there's a lot there).

**Comment by engram-design** [comment]
Ah, now I know the issue. Because SugarCRM is the only 2-legged OAuth provider we support, I had to make some allowances to how the OAuth handshake worked. Unfortunately, this has bled into other providers. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.1",
  "...": "..."
}
```

Then run `composer update`.
